frontend/style/globals.css

```diff
@@ -9,6 +9,7 @@ a {
 
 body,
 html {
+  --text-spacing: 1.2rem 0;
   margin: 0;
   padding: 0;
```

Comment:
Sounds like these few lines can have a huge impact. If you are confident with the change let's go with it

Reply:
I've browse a fair amount of pages and I didn't see any visual regression. This allows to remove custom `p` `margin` which was messing with the default style of `summary-fr` (TOC)